node/pap.md

```diff
@@ -23,35 +23,37 @@ Plug and Play node boxes offer a convenient way to run your own node. Most run o
-2.  [Nodl One](#nodl-one)
-3.  [Nodl Dojo](#nodl-dojo)
-4.  [RaspiBlitz](#raspiblitz)
-5.  [Start9 Server](#start9)
-6.  [Build-A-Node](#build-a-node)
-7.  [The Bitcoin Machine](#the-bitcoin-machine)
-8.  [BTC Cube](#btc-cube)
+1.  [Nodl One](#nodl-one)
+1.  [Nodl Dojo](#nodl-dojo)
+1.  [RaspiBlitz](#raspiblitz)
+1.  [Start9 Server](#start9)
+1.  [Build-A-Node](#build-a-node)
+1.  [The Bitcoin Machine](#the-bitcoin-machine)
+1.  [BTC Cube](#btc-cube)
 
 ***
 
 ### Cost Comparison
 
-| Node                                | Cost                               |
-|-------------------------------------|------------------------------------|
-| [RoninDojo Tanto](#ronindojo-tanto) | $599 (nVME SSD)                    |
-| [myNode One](#mynode-one)           | $339 (HDD)                         |
-|                                     | $419 (SSD)                         |
-| [Nodl One](#nodl-one)               | $499                               |
-|                                     | +$50 w/ kill switch                |
-|                                     | +$50 w/ Dojo support               |
-| [Nodl Dojo](#nodl-dojo)             | $849                               |
-| [RaspiBlitz](#raspiblitz)           | 269€ (2GB)                         |
-|                                     | 299€ (4GB)                         |
-|                                     | 369€ (8GB)                         |
-|                                     | 379€ (4GB) w/ terminal style case  |
-| [Start9](#start9)                   | $499 (8GB Pi), $1799 Pro (x86)     |
-| [Build-A-Node](#build-a-node)       | From $410                          |
-| [The Bitcoin Machine](#the-bitcoin-machine)       | From $199            |
-| [BTC Cube](#btc-cube)       | From $679.95            |
+| Node                                        | Cost                              |
+| ------------------------------------------- | --------------------------------- |
+| [RoninDojo Tanto](#ronindojo-tanto)         | $599 (nVME SSD)                   |
+| [myNode One](#mynode-one)                   | $339 (HDD)                        |
+|                                             | $419 (SSD)                        |
+| [Nodl One](#nodl-one)                       | $499                              |
+|                                             | +$50 w/ kill switch               |
+|                                             | +$50 w/ Dojo support              |
+| [Nodl Dojo](#nodl-dojo)                     | $849                              |
+| [RaspiBlitz](#raspiblitz)                   | 269€ (2GB)                        |
+|                                             | 299€ (4GB)                        |
+|                                             | 369€ (8GB)                        |
+|                                             | 379€ (4GB) w/ terminal style case |
+| [Start9](#start9-server)                    | $199 Lite (8GB Pi)                |
```

Comment:
These 3 offerings are the only thing that actually changed in this table, everything else is alignment (by 'Prettier')